src/Foundation/Site.php

```diff
@@ -178,8 +178,6 @@ protected function getApp()
             $config->set('mail.username', $settings->get('mail_username'));
             $config->set('mail.password', $settings->get('mail_password'));
 
-            $app->register(\Flarum\BusServiceProvider::class);
-
```

Comment:
No replacement?

Reply:
This is a WIP, I was busy stripping down flarum/flarum to get it to work with flarum/core 👍

Reply:
Not sure what you mean, sorry. Does that mean you will revert this before finishing the PR?

Reply:
Fixed. Re-added the ability to auto forward Commands to their respective Handler classes.

Reply:
@franzliedke see https://github.com/flarum/core/pull/1252/commits/72c232d5a388479eaf712939ffeeb19bec848a19